src/common/camera.h

```diff
@@ -39,8 +44,13 @@ class camera {
         // the normalized image-based coordinates of the pixel. Image left is s=0, image right
         // is s=1, image top is t=0, image bottom is t=1.
 
+        // Jitter the sample location of the pixel.
+        s += random_double(-0.5, 0.5) * pixel_delta_x;
+        t += random_double(-0.5, 0.5) * pixel_delta_y;
```

Comment:
This isn't the best place to put this comment, but it's expediant.

Should we be thinking about centering the samples at the center of the pixel? rather than as it is currently at at the edges of pixels?

i.e.
```
        s += 0.5 * pixel_delta_x + random_double(-0.5, 0.5) * pixel_delta_x;
        t += 0.5 * pixel_delta_y + random_double(-0.5, 0.5) * pixel_delta_y;
```
It's strictly more "correct", but it might be additional unnecessary confusion

Reply:
This will get addressed when I rework the pixel math. To recap from our (now lost) Slack conversation, I plan to focus on code that deals with pixel _locations_, and separate code that takes that location and produces pixel samples, not necessarily from a box around the pixel location.